macros/EmbedTest262ReportResultsTable.ejs

```diff
@@ -0,0 +1,12 @@
+<%
+// Embeds a Test262.report results table
+//
+// Parameters:
+//  $0 - An embed URL from test262.report (absolute)
```

Comment:
The reason why https://github.com/mdn/kumascript/blob/master/macros/EmbedGHLiveSample.ejs does use relative URLs is that we don't want to ever allow people to provide arbitrary iframe URLs. Please change this to a relative parameter so that we can make sure people can only ever use test262.report URLs here.

Reply:
That makes perfect sense. I will push a revision shortly. Thanks for the quick review!